alacritty/src/input.rs

```diff
@@ -140,6 +140,15 @@ impl<T: EventListener> Execute<T> for Action {
                     ctx.clear_selection();
                 }
             },
+            #[cfg(not(any(target_os = "macos", windows)))]
+            Action::CopyPrimary => {
+                ctx.copy_selection(ClipboardType::Selection);
+
+                // Clear selection in vi mode for better user feedback
+                if ctx.terminal().mode().contains(TermMode::VI) {
+                    ctx.clear_selection();
+                }
```

Comment:
Small thing, but can we remove the duplication between this block and the `Copy` block above?

Reply:
I don't think so, on other platforms you'll just create separate function to do this, since binding is purely for Linux/BSD. You also can't combine `match` arms due to this, so I'd like to keep things as it is right now.